Modules/Filtering/ImageGrid/include/itkOrientImageFilter.h

```diff
@@ -308,19 +309,13 @@ class ITK_TEMPLATE_EXPORT OrientImageFilter : public ImageToImageFilter<TInputIm
   std::string
   GetMajorAxisFromPatientRelativeDirectionCosine(double x, double y, double z);
 
-  CoordinateOrientationCode m_GivenCoordinateOrientation{
-    SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RIP
-  };
-  CoordinateOrientationCode m_DesiredCoordinateOrientation{
-    SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RIP
-  };
-  bool m_UseImageDirection{ false };
+  CoordinateOrientationCode m_GivenCoordinateOrientation{ AnatomicalOrientation::ToEnum::LPS };
```

Comment:
RIP in From notation should be LSA in To notation, not LPS?

Reply:
No longer including filter in this PR.